code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -830,6 +831,33 @@
 	actions_types = list(/datum/action/item_action/zipper)
 	adjust_flavour = "unzip"
 
+/obj/item/clothing/suit/jacket/cargobomber
+	name = "cargo bomber jacket"
+	desc = "A stylish jacket to keep you warm in the warehouse."
+	icon_state = "bombercargo"
+	item_state = "bombercargo"
+	ignore_suitadjust = 0
+	allowed = list(/obj/item/rcs, /obj/item/clipboard, /obj/item/flashlight, /obj/item/tank/internals/emergency_oxygen, /obj/item/toy, /obj/item/lighter, /obj/item/storage/fancy/cigarettes)
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS
+	cold_protection = UPPER_TORSO|LOWER_TORSO|ARMS
```

Comment:
```suggestion
	body_parts_covered = UPPER_TORSO | LOWER_TORSO | ARMS
	cold_protection = UPPER_TORSO | LOWER_TORSO | ARMS
```
I am aware we dont do this elsewhere, but spacing this kinda thing is good practice.